feat(curriculum): add english challenges (#52343)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/pleasure-to-meet-you.md

---
id: 65528a7c39eb38f51e0e6f7f

> title: "Pleasure to Meet You"

Author: Nice catch. Shoud I click on the Commit suggestion button or is it better another way?

> title: "Pleasure to Meet You"

Reviewer: You can click the commit suggestion button - that will make two commits, one for each suggestion. Otherwise, typically, we like to click on the "files" tab at the top, find the two suggestions and click "add suggestion to batch" next to each of them - then once you've batched all the ones you want to add (those two), click the commit button, and it will only add one commit.

challengeType: 19
dashedName: pleasure-to-meet-you
audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
---

# --description--

> Expressions like `It's a pleasure to meet you` are commonly used when meeting someone for the first time to show politeness and appreciation when meeting someone for the first time.

Reviewer 2: ```suggestion
Expressions like `It's a pleasure to meet you` are commonly used when meeting someone for the first time as a polite way to greet someone.
```
The earlier part of the sentence already has "meeting someone for the first time" and repeating it feels a bit clunky.

# --question--

## --text--

> What does Tom say to express he is happy to meet Maria?

Reviewer 2: ```suggestion
What does Tom say to indicate he is happy to meet Maria?
```
"express" on its own like that seems a little weird, but "show", "indicate", "demonstrate", etc. work well.
